pydy/bodies.py

```diff
@@ -0,0 +1,73 @@
+from sympy.physics.mechanics import RigidBody, Particle
+
+from .force import Force
+
+__all__ = ['Body', 'Ground']
+
+
+class Body(object):
+    """
+
+    Parameters
+    ---------
+    name: string
+        defines the name of the body. It is used as the base for defining body specific
+        properties.
+    (Rest are same as RigidBody but optional)
+    masscenter : Point (optional)
+        The point which represents the center of mass of the rigid body.
+    frame : ReferenceFrame (optional)
+        The ReferenceFrame which the rigid body is fixed in.
+    mass : Sympifyable (optional)
+        The body's mass.
+    inertia : (Dyadic, Point) (optional)
+        The body's inertia about a point; stored in a tuple as shown above.
+
```

Comment:
I did. It creates an instance of particle if only mass and point are provided  and RigidBody otherwise.